net.mkiol.Jupii.yaml

```diff
@@ -0,0 +1,333 @@
+      - --disable-httpupgrade
+      - --enable-shared
+      - --disable-static
+      - --with-pic=yes
+    cleanup:
+      - /bin
+      - /share
+      - /include
+      - /lib/pkgconfig
+      - "*.a"
+      - "*.la"
+    sources:
+      - type: archive
+        url: https://ftpmirror.gnu.org/libmicrohttpd/libmicrohttpd-0.9.77.tar.gz
+        sha256: 9e7023a151120060d2806a6ea4c13ca9933ece4eacfc5c9464d20edddb76b0a0
+
+  - name: npupnp
+    buildsystem: autotools
+    config-opts:
+      - --with-pic=yes
+      - --enable-shared
+      - --disable-static
+    cleanup:
+      - /bin
+      - /share
+      - /include
+      - /lib/pkgconfig
+      - "*.a"
+      - "*.la"
+    sources:
+      - type: archive
+        url: https://www.lesbonscomptes.com/upmpdcli/downloads/libnpupnp-6.1.0.tar.gz
+        sha256: 1e305abf63ac945d9cb4576689670c009a914dc9d05b4c1ed605391e7f6b9719
+
+  - name: upnpp
+    buildsystem: autotools
+    config-opts:
+      - --with-pic=yes
+      - --enable-shared
+      - --disable-static
+    cleanup:
+      - /bin
+      - /share
+      - /include
+      - /lib/pkgconfig
+      - "*.a"
+      - "*.la"
+    sources:
+      - type: archive
+        url: https://www.lesbonscomptes.com/upmpdcli/downloads/libupnpp-0.26.2.tar.gz
+        sha256: d148a522df714ec08b5d03e35c273f340e45eb0d98eb42049247c040814db98f
+
+  - name: pybind11
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DPYBIND11_INSTALL=ON
+      - -DPYBIND11_TEST=OFF
+      - -DPYBIND11_FINDPYTHON=ON
+    cleanup:
+      - /bin
+      - /share
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://github.com/pybind/pybind11/archive/refs/tags/v2.11.1.tar.gz
+        sha256: d475978da0cdc2d43b73f30910786759d593a9d8ee05b1b6846d1eb16c6d2e0c
+
+  - name: x264
+    buildsystem: autotools
+    config-opts:
+      - --enable-pic
+      - --enable-shared
+      - --disable-cli
+    cleanup:
+      - /bin
+      - /share
+      - /lib/pkgconfig
+      - "*.a"
+      - "*.la"
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: 31e19f92f00c7003fa115047ce50978bc98c3a0d
+
+  - name: ffnvc
+    only-arches:
+      - x86_64
+    buildsystem: simple
+    build-commands:
+      - make install PREFIX=${FLATPAK_DEST}
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://github.com/FFmpeg/nv-codec-headers/releases/download/n12.1.14.0/nv-codec-headers-12.1.14.0.tar.gz
+        sha256: 62b30ab37e4e9be0d0c5b37b8fee4b094e38e570984d56e1135a6b6c2c164c9f
+
+  - name: ffmpeg
+    buildsystem: autotools
+    config-opts:
+      - --disable-autodetect
+      - --disable-doc
+      - --disable-programs
+      - --disable-everything
+      - --enable-shared
+      - --disable-static
+      - --enable-nonfree
+      - --enable-gpl
+      - --enable-pic
```

Comment:
```suggestion
      - --disable-static
      - --enable-nonfree
      - --enable-gpl
```

Reply:
almost :)

```
At least one library type must be built.
Specify --enable-static to build the static libraries or --enable-shared to
build the shared libraries as well. To only build the shared libraries specify
--disable-static in addition to --enable-shared.
Error: module ffmpeg: Child process exited with code 1
```

so `--enable-shared` is actually needed.